src/cr/chain_replication_service.cc

```diff
@@ -1,20 +1,21 @@
 #include "cr/chain_replication_service.h"
 #include "hashing_utils.h"
+#include "status_codes.h"
 #include "cr/chain_replication_utils.h"
 namespace smf {
 namespace chains {
 future<smf::rpc_envelope> chain_replication_service::mput(
   smf::rpc_recv_typed_context<tx_multiput> &&puts) {
   smf::rpc_envelope e(nullptr);
-  e.set_status(501); // Not implemented
+  e.set_status(HTTP_STATUS_NOT_IMPLEMENTED); // 501
```

Comment:
It was hard to fully understand what your intent was, I assumed that because the HTTP response codes are well known the goal was not to re-invent that wheel until some point down the line if/when that became useful/necessary.  I'd leave them as HTTP_* for now, just because there are other areas that are more interesting to focus on at this time.